src/widgets/_progressbars.scss

```
@@ -32,14 +32,40 @@ progressbar {
     }
 
     &.horizontal {
-        progress,
         trough {
             @extend %outset-background;
         }
 
         progress {
             margin-left: -1px;
             margin-right: -1px;
+
+            &:not(.pulse) {
+                animation: progress 2s ease infinite;
+                background-size: cover, 48px 100%;
+                background-repeat: no-repeat;
+                background-image:
+                    linear-gradient(
+                        to bottom,
+                        #{'alpha(@highlight_color, 0.2)'},
+                        rgba(white, 0)
+                    ),
+                    linear-gradient(
+                        to right,
+                        rgba(white, 0),
+                        rgba(white, 0.25) 60%,
+                        rgba(white, 0)
+                    );
+
+                &:dir(rtl) {
```

It turns out GTK automatically handles the animation direction for RTL… but not for the inverted-style progress bars… :facepalm: So this should be:

```suggestion
                &.right {
```